[REQUEST][8.18]: Show flyout history for alerts/host/user/events (#6696)
elastic/security-docs: docs/detections/alerts-view-details.asciidoc

From the right panel, you can also:

* Click **Expand details** to open the <<left-panel,left panel>>, which shows more information about sections in the right panel.

> * Click the history icon (image:images/history-icon.png[History icon,15,15]) to display a list of places that you visited from the alert's details flyout. The list shows any rules, alerts, events, users, or hosts who's details you've viewed and can contain up to 10 unique entries. Click an entry in the list to quickly access the object's details.

Reviewer: > The list shows any rules, alerts, events, users, or hosts who's details

@nastasha-solomon Overall looks good to me! I would suggest not list out all the types here. There is also IP flyout, and other teams are developing new flyouts that can show up in the history.

Author: Good thinking! How's this instead:

```suggestion
* Click the history icon (image:images/history-icon.png[History icon,15,15]) to display a list of places that you visited from the alert's details flyout, for example, flyouts for other alerts or users. The list can contain up to 10 unique entries. Click any list entry to quickly access the item's details.
```
